avro-fastserde/src/codegen/java/com/linkedin/avro/fastserde/generated/deserialization/AVRO_1_4/FastStringableTest_javaStringPropertyTest_GenericDeserializer_9185875457190242821_9185875457190242821.java

```diff
@@ -0,0 +1,40 @@
+
+package com.linkedin.avro.fastserde.generated.deserialization.AVRO_1_4;
+
+import java.io.IOException;
+import com.linkedin.avro.fastserde.FastDeserializer;
+import org.apache.avro.Schema;
+import org.apache.avro.generic.IndexedRecord;
+import org.apache.avro.io.Decoder;
+import org.apache.avro.util.Utf8;
+
+public class FastStringableTest_javaStringPropertyTest_GenericDeserializer_9185875457190242821_9185875457190242821
```

Comment:
If we include those new code-gen outputs as part of the commit after the meta-code is already changed, then we can't see if there was any change to this code path, and if there was, what are they. Are you expecting any changes to this code path? If there are no new kinds of changes exposed by those new code-gen outputs, then I'd rather avoid increasing the footprint of checked in code-gen output, but on the other hand, if there are unique changes here, then it would be good to review them. That is why I mentioned in the other comment thread:

> If you choose to add more stuff to the code gen under source control, then you should also split that into an extra commit (one to add the new code gen, one to modify the meta-code, and another one for the modified code gen) otherwise we can't see the diff if it's all in one commit...

I know this requires some git manipulations, but I think it's important since otherwise changes to the meta-code are very hard to review... LMK if you have concerns.